redis-module/src/main/java/team/asd/redis/RedisPoolManager.java

```diff
@@ -0,0 +1,10 @@
+package team.asd.redis;
+
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+
+public class RedisPoolManager {
+	public static JedisPool getPool() {
+		return new JedisPool(new JedisPoolConfig(), "185.126.115.194", 6379, 2000, "testP@ssword");
```

Comment:
Hardcoding sensitive information like the Redis password is a security risk. It's better to externalize such configurations using environment variables or configuration files to enhance security and flexibility